specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Location.json

```diff
@@ -114,37 +114,6 @@
                         }
                     }
                 }
-            },
-            "delete": {
-                "x-ms-examples": {
-                    "Deletes the specified location.": {
-                        "$ref": "./examples/Location/Delete.json"
```

Comment:
can you pls remove the Delete.json in the examples

Reply:
It's done. Thanks.